src/AppBundle/Entity/Form/PersonType.php

```diff
@@ -31,6 +32,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 "label" => "person.label.lastname",
                 "attr" => array("placeholder" => "person.label.lastname")
             ))
+            ->add("avatarFile", FileType::class, array(
+                "label" => "person.label.avatar" 
+            ))
```

Comment:
not required maken door toevoegen van een lijn hier (komma op vorige niet vergeten):
`"required" => false,`

Als je dat bij die hierboven in organisation ook doet dan is dat ook opgelost :-)